services/gateway/app.js

```diff
@@ -69,15 +68,8 @@ broker.createService({
 	name: 'gateway',
 	actions: {
 		spec() { return genDocs(); },
-		status() { return getStatus(); },
-		async ready() {
-			const services = await getReady();
-			// isReady: returns true if any one of service is unavailable
-			const isReady = Object.keys(services.services).some(value => !services.services[value]);
-			if (isReady === true) {
-				return Promise.reject(new MoleculerError('503 Not available', 503, 'ERR_SOMETHING', { services }));
-			} return Promise.resolve(services);
```

Comment:
I think part of this might be still needed. The `503` is not being returned anymore.

Reply:
Now `broker.call` is responsible for calling methods and it throws error `404` or `504` either when services timeout or not found. So, I added `trycatch` which catches error and throw `503` according to our requirement.